### ISSUE TITLE
inconsistent test suites

### ISSUE THREAD

**Issue by namdre** [eclipse/sumo#4573]
```
kobo@nyx:~/dlr/sumo/tests$ WARNING: The test zipper could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/junctions/inspect_selected/type/testsuite.neteditcheckoutput
WARNING: The test create_default_invalid could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/create/testsuite.neteditcheckoutput
WARNING: The test create_manual could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/create/testsuite.neteditcheckoutput
WARNING: The test create_duplicated could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/create/testsuite.neteditcheckoutput
WARNING: The test create_single_crossing could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/create/testsuite.neteditcheckoutput
WARNING: The test create_split_crossing could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/create/testsuite.neteditcheckoutput
WARNING: The test create_priority could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/create/testsuite.neteditcheckoutput
WARNING: The test create_width could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/create/testsuite.neteditcheckoutput
WARNING: The test button_clear could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/create/testsuite.neteditcheckoutput
WARNING: The test button_invert could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/create/testsuite.neteditcheckoutput
WARNING: The test delete could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/testsuite.neteditcheckoutput
WARNING: The test inspect could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/testsuite.neteditcheckoutput
WARNING: The test inspect_selected could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/testsuite.neteditcheckoutput
WARNING: The test load could not be found.
Please check the file at /home/kobo/dlr/sumo/tests/netedit/netElements/connections/testsuite.neteditcheckoutput

```

**Comment by namdre** [comment]
most likely a consequence of renaming tests

**Comment by palvarezlopez** [comment]
Fixed in 815b90df5405817e73ab99bbdf50e3093c43074e
